src/EditorFeatures/CSharpTest2/Recommendations/ClassKeywordRecommenderTests.cs

```diff
@@ -280,6 +280,13 @@ public async Task TestAfterPublic()
 @"public $$");
         }
 
+        [Fact, WorkItem(66319, "https://github.com/dotnet/roslyn/issues/66319")]
+        public async Task TestAfterFile()
+        {
+            await VerifyKeywordAsync(SourceCodeKind.Regular,
+@"file $$");
+        }
+
```

Comment:
Needs a test for record recommender as well.

Reply:
Sounds good!